opendevin/sandbox/sandbox.py

```diff
@@ -0,0 +1,182 @@
+import os
```

Comment:
renamed this file because `import docker` tries to import this instead of the SDK.